about.html

```diff
@@ -0,0 +1,111 @@
+        </ul>
+    </nav>
+    <section class="section-one padding-x">
+        <h3>"Hallo sharing World"</h3>
+        <h1>CREATIVE COMMONS <br> GLOBAL SUBMMIT 2015</h1>
+        <div class="ipsum">
+            <p>It has survived not only five 
+                centuries, but also the leap into electronic typesetting, remaining essentially unchanged. </p><p>
+                It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum 
+                passages, and more recently with desktop publishing software like Aldus PageMaker including
+                versions of Lorem Ipsum.</p>
+        </div>
+        <div>
+            <p class="p">Please contact per email for any further queston about about CC Global summit 2015! </p>
+            <p class="email">ccsummit2015@cckorea.org</p>
+        </div>
+    </section>
+    <section class="section-two padding-x">
+        <h1>Creative common Global summit 2015 logo</h1>
+        <span class="red_underline"></span>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard 
+            dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a 
+            type specimen book. It has survived not only five centuries, but also the leap into electronic 
+            typesetting, remaining essentially unchanged.
+        </p>
+        <div>
+            <img class="logo" src="./imgs/icons/logo.png" alt="Logo Images">
+        </div>
+    </section>
+    <section class="section-three padding-x">
+        <h1>See the pass Global cc summits</h1>
+        <span class="red_underline"></span>
+        <p>
+            Tke a look at past two cc global summit which took place in benousalians and in warsaws
+        </p>
+        <div>
+            <section class="image-1"><p><b>2015</b>Event on last year while students was celebrationg their day.</p></section>
+            <section class="image-2"><p><b>2018</b> on last year while students was celebrationg their day.</p></section>
+        </div>
+    </section>
+    <section class="parnter-section-about padding-x">
+        <h2>Partner</h2>
+        <span class="red_underline"></span>
+        <div class="partner-content">
+            <div>
+                <img src="./imgs/background/dtu.png" alt="">
+                <p>DTU</p>
+            </div>
+            <div>
+                <img src="./imgs/background/uog.png" alt="">
+                <p>UoG</p>
+            </div>
+            <div>
+                <img src="./imgs/background/Arba_Minch_University.png" alt="">
+                <p>AMU</p>
+            </div>
+            <div>
+                <img src="./imgs/background/MoE.png" alt="">
+                <p>MoE</p>
+            </div>
+            <div>
+                <img src="./imgs/background/dtu.png" alt="">
+                <p>BDU</p>
+            </div>
+        </div>
```

Comment:
Please add alt text for **all** images in your website. This will improve your website's accessibility. Please don't forget to update the images in index.html